app/controllers/application_controller.rb

```diff
@@ -14,6 +14,12 @@ class ApplicationController < ActionController::Base
     redirect_to login_url, :alert => exception.message
   end
 
+  def reload_page
+    respond_to do |format|
+      format.js { render inline: 'location.reload();' }
+    end
+  end
```

Comment:
i want you to remove this method there is no place for this stupid thing. We should only use `redirect_to` with flash messages and not get back to js reloading it's not good.